### ISSUE TITLE
custom areas.yaml

### ISSUE THREAD

**Issue by hproe** [pytroll/satpy#1492]
## Feature Request

While where to place custom composites and enhancements is fully documented in the relative chapters of the satpy doc. Also, the possibility of creating custom areas is briefly discussed, but nothing is said (?) where to place such a custom areas.yaml. Though quite obvious, it should be mentioned explicitly that the custom areas.yaml has to be placed where the environment variable PPP_CONFIG_DIR is pointing to. I suggest to add this information under "Create custom area definition".

**Comment by djhoese** [comment]
Agreed. I am working on a restructuring of how Satpy handles these configuration directories so maybe I can include this information when I finish that. Thanks for filing the issue.
